http/vibe/http/server.d

```diff
@@ -633,6 +639,15 @@ final class HTTPServerSettings {
 	*/
 	string hostName;
 
+	/** Provides a way to ban and unban network addresses and reduce the
+		impact of DOS attack.
+
+		If any of the ban managers bans an address, then all incoming requests
+		from that address will be rejected.
+	*/
+	alias IsBannedDg = bool delegate (in NetworkAddress) @safe nothrow;
+	IsBannedDg isBannedDg;
```

Comment:
```suggestion
	*/
	IsBannedDg isBannedDg;
	
	/// Type of delegate accepted ffor `isBannedDg`
	alias IsBannedDg = bool delegate (in NetworkAddress) @safe nothrow;
```

Otherwise the DDOC will be wrong.